conf/rabbitmq/definitions.json

```diff
@@ -55,7 +55,7 @@
     {
       "hashing_algorithm": "rabbit_password_hashing_sha256",
       "name": "admin",
-      "password_hash": "",
+      "password_hash": "+MYeOpbWlzJ5k+xCEylpDDL5IzJdUttN8HNqGFVc/qNwOp5n",
```

Comment:
Please delete this file. It is not necessary @vasanth096

Reply:
Deleted